model/src/main/java/org/mskcc/cmo/metadb/model/MetaDbRequest.java

```diff
@@ -43,6 +43,8 @@
     protected String libraryType;
     protected List<RequestSample> requestSamples;
     protected List<String> pooledNormals;
+    protected boolean isCmoRequest;
```

Comment:
`isCmoRequest` is the field being removed and `cmoRequest` is what's staying I thought

Reply:
you're right, my bad